lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -902,7 +902,7 @@ sub dependencies ($self) {
 sub investigate {
     my ($self) = @_;
     return $self->reply->not_found unless my $job = $self->_get_current_job;
-    my $investigation = $job->investigate;
+    my $investigation = $job->investigate(git_limit => 200);
```

Comment:
It would be nice if this value could come from the config file or from an env variable.

Reply:
It is now configurable